product_specifications/34DbH5/igsn.feature

```diff
@@ -0,0 +1,89 @@
+
+Feature: Repository exclusive for IGSN-DOIs
+
+        Scenario: Creating a IGSN Catalogue in Fabrica
+            Given that a user is creating a repository in Fabrica
+             When the user chooses the repository type
+             Then the  repository type field should show the 'IGSN Catalogue' option
+             When the user selects 'IGSN Catalogue'
+             Then a info message should be displayed reading: "This repository will only be able to mind IGSN-DOIs."
+             When clicking create a new repository with type 'IGSN Catalogue' should be created.
+ 
+        Scenario: Editing a IGSN Catalogue in Fabrica
+            Given the repository is type IGSN Catalogue
+             When the user is editing in the IGSN Catalogue edit form
+             Then the repository type field should be disabled.
+              And a tooltip should be shown when hovering over the field that read "IGSN Catalogues cannot be edited"
+
+        Scenario: Editing a repository in Fabrica
```

Comment:
Existing repositories type can't be changed to IGSN Catalogue?

Reply:
correct. Same reason that above.